src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckJavaAgent.kt

```diff
@@ -178,16 +182,11 @@ internal object LincheckJavaAgent {
                 if (!INSTRUMENT_ALL_CLASSES && canonicalClassName !in instrumentedClasses) {
                     return@mapNotNull null
                 }
-                // For each class, retrieve its original bytecode.
-                val bytes = nonTransformedClasses[clazz.name]
-                check(bytes != null) {
-                    "Original bytecode for the transformed class ${clazz.name} is missing!"
-                }
-                ClassDefinition(clazz, bytes)
+                clazz
             }
```

Comment:
Could you please then change `mapNotNull` to `filter` and also rename `classDefinitions` into just `classes`.

Reply:
done